### PR TITLE
stop bazel after building

### PR DESCRIPTION
**Pull request by BenTheElder**
stopgap measure to hopefully help reduce pressure prow.k8s.io

cc @dims @liggitt

### CODE REVIEW

File: hack/ci/e2e-k8s.sh [hunk 1]
```diff
@@ -49,6 +49,9 @@ build_with_bazel() {
   # make sure we have e2e requirements
   bazel build //cmd/kubectl //test/e2e:e2e.test //vendor/github.com/onsi/ginkgo/ginkgo
 
+  # free up memory by terminating bazel
+  bazel shutdown
```
Comment (dims) [inline hack/ci/e2e-k8s.sh]:
```
bazel shutdown || true
pkill ^bazel || true
```

Comment (BenTheElder) [reply hack/ci/e2e-k8s.sh]: done

**Pull request merged**